src/views/Results.svelte

```diff
@@ -99,6 +113,33 @@
 <script lang="ts">
     export let cols: Cols = 6;
 
+    let total = 0;
+    $: (async () => {
+        const fetchedProviders = await Promise.all($results);
+        const filteredProviders = fetchedProviders.filter(
+            ([apis, provider]) =>
+                apis &&
+                apis.some((a) => !(a instanceof Error) && a?.data.length)
+        );
+        const returned = filteredProviders.reduce((acc, [apis, provider]) => {
+            acc = acc.length
+                ? [...acc, apis[0].meta.data_returned]
+                : [apis[0].meta.data_returned];
+            return acc;
+        }, []);
+        total = returned.length && Math.max(...returned) / 10;
+        console.log(
+            filteredProviders,
+            returned,
+            returned.length && Math.max(...returned)
+        );
+    })();
+
+    function setPage(page: Param) {
+        page = page === 0 ? 1 : page;
```

Comment:
@Valexr so the pages for **zero** and **one** are the same? :thinking:

Reply:
Oh... this's a long story )))
- in Optimade Apis pages starts from zero - `page_number=0`
- in Pagination pages starts from one - `page = 1`
- function setPage() just transforming `0 -> 1` for Pagination component
- but in url we have `page_number=0`
- but `page = 1`

Reply:
really -  in optimade.science `0 === 1` same value for API request... why it's don't working in GUI...🤔

Reply:
I believe this complexity should be hidden inside the optimade client. It's not so necessary to do that right now, but it would be better to work with normal page numbers starting from 1 always. So, we can just make this transformation inside optimade client, eg:


```
  getStructuresAll(providerIds: string[], filter: string = '', page: number = 1, limit: number, batch: boolean = true) {
        page -= 1; // we recieve always valid page starting from 1, but transform it to optimade formart starting form 0
        ...
  }
```

In apps, we should always use normal pages values, in code and in URLs, starting from 1.

Reply:
already in https://github.com/tilde-lab/optimade-client/blob/ba22136527064a690532c4394f18799dceaf7f73/src/index.ts#L74